install.sh

```diff
@@ -0,0 +1,62 @@
+set -e
+
+if [ -z "$(uname | grep Darwin)" ]; then
+  OS=linux
+  set -x
+else
+  OS=osx
+fi
+
+echo "Checking if dbdeployer is installed"
+if ! [ -x "$(command -v dbdeployer)" ]; then
+  echo "Not installed...starting install"
+  VERSION=1.8.0
+  origin=https://github.com/datacharmer/dbdeployer/releases/download/$VERSION
+  filename=dbdeployer-$VERSION.$OS
+  wget -q $origin/$filename.tar.gz
+  tar -xzf $filename.tar.gz
+  chmod +x $filename
+  sudo mv $filename /usr/local/bin/dbdeployer
```

Comment:
Is there a reason why we don't use docker-compose [like we do in Ghostferry](https://github.com/Shopify/ghostferry/blob/master/docker-compose.yml)? This would save all this code of having to download a custom tool and mysql.

I'm somewhat sketched out that this script would install something into my /usr and that this is a hard requirement for running integration tests. If we need to stick with this setup for some requirement and can't use docker-compose, we could unpack this into a vendor directory or even just distribute the binary with this repo and call the executable via relative paths.

Reply:
I have to take responsibility for using dbdeployer in this use case. I suggested Brian to go with it.

IIRC, configuring a master-slave replication arrangement with docker compose was not solved out of the box.

Docker compose may be good enough for test suites, but I want us to get familiar with a tool that is going to allow us to start and stop mysql replicated clusters at will and easily in our machines.

Doing that with docker compose just to test something, is far from trivial.